docs/conf.py

```diff
@@ -195,14 +195,14 @@
 
 # -- Options for LaTeX output ---------------------------------------------
 
-latex_elements = {
-    # The paper size ('letterpaper' or 'a4paper').
-    #'papersize': 'letterpaper',
-    # The font size ('10pt', '11pt' or '12pt').
-    #'pointsize': '10pt',
-    # Additional stuff for the LaTeX preamble.
-    #'preamble': '',
-}
+# latex_elements = {
```

Comment:
tiny nit... if we're adding this to the template, and essentially to all repos eventually, this could be more readable (and turned on with just one command to remove commented out lines) as:

```py
# latex_elements = {
#     # The paper size ('letterpaper' or 'a4paper').
#     'papersize': 'letterpaper',
#     # The font size ('10pt', '11pt' or '12pt').
#     'pointsize': '10pt',
#     # Additional stuff for the LaTeX preamble.
#     'preamble': '',
# }
```

Reply:
Linting had de-indented it, so I figured that was just how it wanted it. But it's ok with it now :shrug: